tests/rbd_mirror/test_rbd_non-default_to_default_namespace_mirroring.py

```diff
@@ -0,0 +1,287 @@
+            remote_namespace = pool_config.get("remote_namespace")
+
+            for image, image_config_val in image_config.items():
+                image_enable_config = {
+                    "pool": pool,
+                    "image": image,
+                    "mirrormode": "snapshot",
+                    "namespace": namespace,
+                    "remote_namespace": remote_namespace,
+                }
+                # Enable snapshot mode mirroring on images of the namespace
+                enable_image_mirroring(pri_config, sec_config, **image_enable_config)
+                # Verify image mirroring status on primary cluster
+                wait_for_status(
+                    rbd=rbd_primary,
+                    cluster_name=pri_config.get("cluster").name,
+                    imagespec=f"{pool}/{namespace}/{image}",
+                    state_pattern="up+stopped",
+                )
+                # Verify image mirroring status on secondary cluster
+                wait_for_status(
+                    rbd=rbd_secondary,
+                    cluster_name=sec_config.get("cluster").name,
+                    imagespec=f"{pool}/{image}",
+                    state_pattern="up+replaying",
+                )
+                if image_config_val.get(
+                    "snap_schedule_levels"
+                ) and image_config_val.get("snap_schedule_intervals"):
+                    for level, interval in zip(
+                        image_config_val["snap_schedule_levels"],
+                        image_config_val["snap_schedule_intervals"],
+                    ):
+                        snap_schedule_config = {
+                            "pool": pool,
+                            "image": image,
+                            "level": level,
+                            "interval": interval,
+                            "namespace": namespace,
+                        }
+                        # Adding snapshot schedules to the images in namespace
+                        out, err = add_snapshot_scheduling(
+                            rbd_primary, **snap_schedule_config
+                        )
+                        if err:
+                            raise Exception(
+                                "Adding snapshot schedule failed with error " + err
+                            )
+                    # Verify snapshot schedules are effective on the namespaces
+                    verify_namespace_snapshot_schedule(
+                        rbd_primary, pool, namespace, interval=interval, image=image
+                    )
+
+                pri_image_spec = f"{pool}/{namespace}/{image}"
+                sec_image_spec = f"{pool}/{image}"
+                # Write data on the primary image
+                bench_kw = kw.get("config").get("io")
+                bench_kw.update({"image-spec": pri_image_spec})
+                out, err = rbd_primary.bench(**bench_kw)
```

Comment:
pls run IO with `FIO config` along with `disk mount` because that is the actual customer use case

Reply:
for manual execution anything is fine hence that way in polarion, but in automation, we should follow the best approach also we need to cover ODF workload
https://github.com/red-hat-storage/cephci/pull/4672/files#diff-f8628706ebf5a81881d8b8e4a3e7587d10ae86f7690f476bdd5158ab44125bf3R50
please run this IO it's not a big change.